File: python/fizzbuzz.py
#!/usr/bin/env python3

MODULUS_WORDS = {15: "FizzBuzz", 3: "Fizz", 5: "Buzz"}

for i in range(1, 101):
    output = f"{i}"

    for modulus in MODULUS_WORDS:
        if i % modulus == 0:
            output += f": {MODULUS_WORDS[modulus]}"
            break

    print(output)

File: javascript/FizzBuzz.js
for (number in [...Array(101).keys()]) {
    if (number % 5 == 0 && number % 3 == 0) {
        console.log(`${number}: FizzBuzz`);
    } else if (number % 3 == 0) {
        console.log(`${number}: Fizz`);
    } else if (number % 5 == 0) {
        console.log(`${number}: Buzz`);
    } else {
        console.log(number);
    }
}